ddtrace/appsec/_common_module_patches.py

```diff
@@ -61,7 +61,9 @@ def wrapped_read_F3E51D71B4EC16EF(original_read_callable, instance, args, kwargs
     wrapper for _io.BytesIO and _io.StringIO read function
     """
     result = original_read_callable(*args, **kwargs)
-    if asm_config._iast_enabled:
+    from ddtrace.appsec._iast._utils import _is_iast_enabled
+
+    if _is_iast_enabled():
```

Comment:
It's indeed out of date, I think it's safe to just revert this

```suggestion
    if asm_config._iast_enabled:
```

Reply:
@gnufede this is actually a change I made to get around the fact that `_taint_tracking/__init__.py` unconditionally uses names that are conditionally imported.